sources/manager_github.py

```
@@ -112,44 +112,56 @@ def _copy_file_and_add_to_repo(src_path: str):
         GitHubManager.REPO.git.add(dst_path)
 
     @staticmethod
-    def update_readme(stats: str) -> None:
+    def update_readme(stats: str, section_name: str = None) -> None:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)